src/installations/index.ts

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { app } from '../firebase-namespace-api';
+
+/**
+ * Gets the {@link installations.Installations `Installations`} service for the
+ * default app or a given app.
+ *
+ * `admin.installations()` can be called with no arguments to access the default
+ * app's {@link installations.Installations `Installations`} service or as
+ * `admin.installations(app)` to access the
+ * {@link installations.Installations `Installations`} service associated with a
+ * specific app.
+ *
+ * @example
+ * ```javascript
+ * // Get the Installations service for the default app
+ * var defaultInstallations = admin.installations();
+ * ```
+ *
+ * @example
+ * ```javascript
+ * // Get the Installations service for a given app
+ * var otherInstallations = admin.installations(otherApp);
+ *```
+ *
+ * @param app Optional app whose `Installations` service to
+ *   return. If not provided, the default `Installations` service will be
+ *   returned.
```

Comment:
"will be returned" is one of the cases where "is returned" is preferable.

Suggest checking all instances of "will" to see if present tense works.  If it does, it is preferred.

Reply:
Done